Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -14,7 +14,7 @@ class MSMainViewController: UITableViewController, AppCenterProtocol {
     
     static let allValues = [AppCenter, OneCollector, Both, None, Skip]
   }
-
+  
```

Comment:
I don't think this change is necessary.

Reply:
OK, Jason will revert the last changes and create new branch based on this branch